### PR TITLE
[reggen] Add tentative list of assets and cm types

### PR DESCRIPTION
**Pull request by msfschaffner**
This is in preparation for the first D2S reviews (follow up from #9240).
The names are tentative and have been proposed by @tjaychen.
The idea is to start with this initial set and refine as needed.

Signed-off-by: Michael Schaffner <msf@opentitan.org>

### CODE REVIEW

File: util/reggen/countermeasure.py [hunk 1]
```diff
@@ -2,25 +2,54 @@
...
 
+import re
 from typing import Dict, List
 
 from .lib import check_keys, check_str, check_list
 
-# TO BE DEFINED
 CM_ASSETS = [
-    'FOO'
+    'KEY',
```
Comment (tjaychen) [inline util/reggen/countermeasure.py:11]: nice! 
in case the intent of these names aren't clear, where do you think those should be documented?

**Review by tjaychen — APPROVED** [review]

File: util/reggen/countermeasure.py (hunk 1 above)
Comment (msfschaffner) [reply util/reggen/countermeasure.py:11]: I would say we add it to the comportable IP spec or similar...

Comment (msfschaffner) [reply util/reggen/countermeasure.py:11]: We can do that once the D2S checklist is finalized.

Comment (tjaychen) [reply util/reggen/countermeasure.py:11]: that makes sense.

**Pull request merged**